recipes/example/meta.yaml

```diff
@@ -60,6 +62,7 @@ about:
 
 extra:
   recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
```

Comment:
Note to others: Same as above. Comes from PR ( https://github.com/conda-forge/staged-recipes/pull/1632 ) already merged into `master`.